aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -895,6 +914,11 @@ module aptos_framework::stake {
         let len = vector::length(&validator_set.active_validators);
         while (i < len) {
             let validator = vector::borrow(&validator_set.active_validators, i);
+            spec {
+                assume exists<StakePool>(validator.addr);
```

Comment:
Is there a better way to say assume initialize_... has been called?

Reply:
For that, we now use the spec function `spec_is_initialized_validator`.